specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/2020-01-15-generalAvailability/CommunicationIdentity.json

```diff
@@ -0,0 +1,264 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CommunicationIdentityClient",
+    "description": "Azure Communication Identity Service",
+    "contact": {
+      "email": "acsdevexdisc@microsoft.com"
+    },
+    "version": "2020-01-15-generalAvailability"
+  },
+  "paths": {
+    "/identities": {
```

Comment:
We still have not solved the idempotency problem here.

Reply:
Is it a reminder now? Or is here anything I can do now?

Reply:
If this is solely for the next preview, then idempotency comment can be ignored. But if we are thinking about our final APIs, then we need to address it.

Reply:
I thought final API should be renamed from preview to something else right? So we are going to create another change before GA at least for it.. Is my assumption  correct?

Reply:
After the discussion with our PM we do not consider this as GA ready .. this are changes after review from September, otherwise we have problems to step forward.